saleor/checkout/models.py

```diff
@@ -49,6 +49,7 @@ class CheckoutShippingMethod(models.Model):
     name = models.CharField(max_length=255)
     description = models.TextField(null=True, blank=True)
 
+    price = MoneyField(amount_field="price_amount", currency_field="currency")
     price_amount = models.DecimalField(max_digits=10, decimal_places=2)
```

Comment:
IMO we should use `settings.DEFAULT_MAX_DIGITS` and `settings.DEFAULT_DECIMAL_PLACES` instead of values here.